<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendLaterWidget</class>
 <widget class="QWidget" name="SendLaterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>85</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mLabel">
     <property name="text">
      <string>Send at:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="SendLater::SendLaterTimeDateWidget" name="mDateTime" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="mRecurrence">
     <property name="text">
      <string>Repeat every:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="mRecurrenceValue">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>365</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="KComboBox" name="mRecurrenceComboBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>SendLater::SendLaterTimeDateWidget</class>
   <extends>QWidget</extends>
   <header>sendlatertimedatewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
